lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -76,6 +76,7 @@
 import static java.lang.Math.max;
```

Comment:
Would be great if it could wait for @raunaqmorarka since he touches `ParquetReader` in https://github.com/trinodb/trino/pull/13246 too